deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore/deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
                   - key: node.kubernetes.io/instance-type
                     operator: In
                     values:
-                      - r6a.xlarge
+                      - r5n.2xlarge
```

Comment:
Two questions:
* Are the existing node groups for this instance type in the right AZ?
* What is the max number of instances in the corresponding node group and does it have headroom to grow in order to fit the new node?

Reply:
* new node group is in [us-east-2c](https://github.com/ipni/storetheindex/blob/main/deploy/infrastructure/dev/us-east-2/eks.tf#L91), same to what was before
* trhe max size is 3 and it's currently running 2 instances - ber and cali.

Reply:
Actually ber runs in (2b)[https://github.com/ipni/storetheindex/blob/main/deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/ber/deployment.yaml#L33]. So it's just one node there - cali. Do you still want to bump to 5?

Reply:
Good not needed. 

The thing to point out here is that rolling deployment starts up a pod, wait for it to be ready before tearing down the old one. That's why extra headroom is needed in ASG so that new worker nodes can be started to run the deployment. 

It all cleans itself up back to minimal number of nodes.